docs/developing.md

```diff
@@ -0,0 +1,58 @@
+# Frontend Shared Package
+
+The `frontend-shared` package is a library of shared utilities, components and styles common to
+multiple Hypothesis front-end applications.
+
+## Client setup
+Follow these steps to set up ``frontend-shared`` in the client repository.
```

Comment:
Can you add a blank line after each heading. I just find this easier to read and more conventional in our markdown files.